### PR TITLE
fix(manager) : Changes in web property instruction page.

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1) Ephemeral environment shows up for new property with no spas and instructions to create spas are visible under SPAs tab.
2) Environment names are converted to lowercase, while creating new environments and while creating new property
3) Resizing of the search box for filtering spas
4) code linting

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

Pasted Graphic

Pasted Graphic 3

Pasted Graphic 2

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by soumyadip007 — APPROVED** [review]
LGTM

**Review by hybridx — APPROVED** [review]
@nikhitamore can you please bump up the version? Rest looks good to me.

**Review by hybridx — APPROVED** [review]

File: packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx [hunk 1]
```diff
@@ -124,263 +124,260 @@ export const WebPropertyDetailPage = (): JSX.Element => {
...
+                          <SelectOption key={`${envName} + ${index + 1}`} value={envName} />
+                        ))}
+                      </Select>
+                    </SplitItem>
+                  </Split>
+                  {filterByEnv === 'Select Enviroment' || filterByEnv === '' ? (
+                    <p />
+                  ) : (
+                    <div
+                      style={{
```
Comment (akhilmhdh) [inline packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx:182]: @kotwani2883 noticed you added this part in previous commit. 

@kotwani2883 @ayushsatyam146 Kindly go thorugh pfe patternfly and next time try to use that. Let's avoid putting style like this as much as possible.

**Review by akhilmhdh — APPROVED** [review]

File: packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx (hunk 1 above)
Comment (kotwani2883) [reply packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx:182]: Noted @akhilmhdh will change and raise the PR

**Pull request merged**